npc/re/merchants/rgsr_in.txt

```diff
@@ -0,0 +1,451 @@
+	mes "[Torus]";
+	mes "Huh? Do you have anyone else to see on our team other than the team leader?";
+	mes "Then that's me~";
+	mes "I am in charge of Enchants.";
+	mes "Let's see what you got?";
+	next;
+	if (select( "Modified OS Weapon Enchant", "Quit" ) == 2) {
+		mes "[Torus]";
+		mes "Come back whenever you need it.";
+		mes "And we need the research.";
+		close;
+	}
+	mes "[Torus]";
+	mes "Where...";
+	close2;
+	item_enchant(33);
+	end;
+
+OnInit:
+	setunittitle( getnpcid (0), "Team Matnard" );
+	end;
+}
+
+rgsr_in,115,169,5	script	Weapon Researcher#rg02	MD_ED_M_SCIENCE,{
+	mes "[Frasa]";
+	mes "Let's see... You're a guest.";
+	mes "I like people who are sure what they want.";
+	mes "Modifiable items can also be read as <TIPBOX>[List]<INFO>9163</INFO></TIPBOX>.";
+	mes "I will try to answer any questions you have.";
+	next;
+	switch( select( "Thanos Weapons", "OS Weapons", "Quit" ) ) {
+	case 1:
+		mes "[Frasa]";
+		mes "^0000FFThanos weapon^000000 mods have lower and intermediate levels in the modification method.";
+		next;
+		switch( select( "Lower mods", "Intermediate mods", "Quit" ) ) {
+		case 1:
+			mes "[Frasa]";
+			mes "Thanos weapon lower modification.";
+			mes "^FF0000Smelting level is reduced by 5^000000, but consumes less materials.";
+			mes "And for mods, the weapon needs +9 or more smelting levels.";
+			close2;
+			item_reform("Thanos_W_Reform_1");
+			end;
+		case 2:
+			mes "[Frasa]";
+			mes "Thanos Weapon intermediate modification.";
+			mes "^FF0000Smelting level is reduced by 1^000000, but consumes more material.";
+			mes "And for mods, the weapon needs +9 or more smelting levels.";
+			close2;
+			item_reform("Thanos_W_Reform_2");
+			end;
+		case 3:
+			mes "[Frasa]";
+			mes "If you need mods, come back.";
+			mes "You are welcome any time you help us with our research.";
+			close;
+		}
+		end;
+	case 2:
+		mes "[Frasa]";
+		mes "^0000FFOS Weapons^000000...";
+		mes "For modification, the smelting level must be +7 or more.";
+		mes "Smelting level does not decrease.";
+		mes "Did you bring the stuff?";
+		next;
+		if (select( "Convert", "Quit" ) == 2) {
+			mes "[Frasa]";
+			mes "If you need mods, come back.";
+			mes "You are welcome any time you help us with our research.";
+			close;
+		}
+		mes "[Frasa]";
+		mes "Let's see what you brought.";
+		close2;
+		item_reform("OS_Weapon_Reform");
+		end;
+	case 3:
+		mes "[Frasa]";
+		mes "If you need mods, come back.";
+		mes "You are welcome any time you help us with our research.";
+		close;
+	}
+	end;
+
+OnInit:
+	setunittitle( getnpcid (0), "Team Frasa");
+	end;
+}
+
+rgsr_in,115,180,5	script	Armor Researcher#rg02	MD_ED_M_SCIENCE,{
+	mes "[Algebra]";
+	mes "Oh, at best, we've come here because our team's research is still short on experiments.";
+	mes "Next time you get a little more improvement and get the permission, then let's proceed.";
+	mes "Okay?";
+	npctalk "Grimace.", "", bc_self;
+	close;
+
+OnInit:
+	setunittitle( getnpcid (0), "Team Matnard" );
```

Comment:
Its only a guess, but could it be that this should be "Team Mathematik"?
Both NPC names are related to mathematics (Algebra and Torus).

Reply:
Not sure, it could also be `Atnad` like in the ep15
https://irowiki.org/wiki/To_Phantasmagorika!